streams/src/main/java/org/apache/kafka/streams/kstream/internals/InternalStreamsBuilder.java

```diff
@@ -388,14 +391,15 @@ private void maybeUpdateKeyChangingRepartitionNodeMap() {
 
     @SuppressWarnings("unchecked")
     private OptimizableRepartitionNode createRepartitionNode(final String name,
```

Comment:
Would you be ok with renaming "name" to "nodeName" to disambiguate?

Reply:
ack